go.mod

```diff
@@ -9,3 +9,5 @@ require (
 	github.com/pkg/errors v0.9.1
 	golang.org/x/net v0.0.0-20210423184538-5f58ad60dda6
 )
+
+require go.uber.org/multierr v1.6.0
```

Comment:
It didn't. I'm assuming that's because it [already exists](https://github.com/edaniels/zeroconf/blob/register_dynamic_fork/go.sum#L353-L354) in `go.sum`?

Reply:
yep